adapters/qq/src/utils.ts

```diff
@@ -34,8 +34,6 @@ export function decodeGroupMessage(
   payload: Universal.MessageLike = message,
 ) {
   message.id = data.id
-  const date = data.timestamp.slice(0, data.timestamp.indexOf('m=')).trim().replace(/\+(\d{4}) CST/, 'GMT+$1')
-  message.timestamp = new Date(date).valueOf()
```

Comment:
Why moving these two lines to below?

Reply:
referenced https://github.com/satorijs/satori/blob/8d1a614daaf7eeb0ddad73009d4971cca5c8e368/adapters/qq/src/utils.ts#L88